daemon/cmd/kube_proxy_replacement.go

```diff
@@ -462,15 +470,19 @@ func finishKubeProxyReplacementInit() error {
 		// All cases below still need to be implemented ...
 		case option.Config.EnableEndpointRoutes:
 			msg = fmt.Sprintf("BPF host routing is currently not supported with %s.", option.EnableEndpointRoutes)
-		case !mac.HaveMACAddrs(option.Config.GetDevices()):
-			msg = "BPF host routing is currently not supported with devices without L2 addr."
-		case option.Config.EnableWireguard:
-			msg = fmt.Sprintf("BPF host routing is currently not compatible with Wireguard (--%s).", option.EnableWireguard)
+		//
+		case !mac.HaveMACAddrs(option.Config.GetDevices()) || option.Config.EnableWireguard:
+			if kernelVersion, err := version.GetKernelVersion(); err == nil {
```

Comment:
We don't handle the `err` returned here if it's not nil. Is there any way to log it?